load_confounds/parser.py

```diff
@@ -345,47 +251,49 @@ def _load_motion(self, confounds_raw):
 
     def _load_high_pass(self, confounds_raw):
         """Load the high pass filter regressors."""
-        high_pass_params = _find_confounds(confounds_raw, ["cosine"])
+        high_pass_params = cf._find_confounds(confounds_raw, ["cosine"])
         return confounds_raw[high_pass_params]
 
     def _load_wm_csf(self, confounds_raw):
         """Load the regressors derived from the white matter and CSF masks."""
         wm_csf_params = cf._add_suffix(["csf", "white_matter"], self.wm_csf)
-        _check_params(confounds_raw, wm_csf_params)
+        cf._check_params(confounds_raw, wm_csf_params)
         return confounds_raw[wm_csf_params]
 
     def _load_global(self, confounds_raw):
         """Load the regressors derived from the global signal."""
         global_params = cf._add_suffix(["global_signal"], self.global_signal)
-        _check_params(confounds_raw, global_params)
+        cf._check_params(confounds_raw, global_params)
         return confounds_raw[global_params]
 
     def _load_compcor(self, confounds_raw):
         """Load compcor regressors."""
         if self.compcor == "anat":
```

Comment:
I think `_load_compcor` need some check for invalid inputs. Can extract the if statements on `self.compcor` and `self.acompcor_combined` to python dictionaries, use that to catch errors, and migrate to `confounds.py`. I can do a PR to this branch .

Reply:
Thanks a lot for the PR, beautifully done!

Reply:
Agreed. Thanks for the PR which addresses this issue.